appveyor.yml

```diff
@@ -14,12 +14,12 @@ skip_tags: true
 #faster cloning
 clone_depth: 1
 
-# Install the latest nightly of ChefDK
+# Install the latest nightly of Chef Workstation
 install:
-  - ps: (& cmd /c); iex (irm https://omnitruck.chef.io/install.ps1); Install-Project -Project chefdk -channel current
+  - ps: (& cmd /c); iex (irm https://omnitruck.chef.io/install.ps1); Install-Project -Project chef-workstation -channel current
   - ps: 'Get-CimInstance win32_operatingsystem -Property Caption, OSArchitecture, Version | fl Caption, OSArchitecture, Version'
   - ps: $PSVersionTable
-  - c:\opscode\chefdk\bin\chef.bat exec ruby --version
+  - c:\opscode\chefworkstation\bin\chef.exe exec ruby --version
```

Comment:
chef-workstation

Reply:
I had that initially, but I think it was .bat vs. .exe when we switched from ruby to go there that broke me. Of course the one time I need my windows laptop the battery was dead.